lib/union/index.js

```diff
@@ -1,9 +1,9 @@
 import _curry2 from '../_internal/_curry2'
 
 export default _curry2(function union(arr1, arr2) {
-  return arr2.reduce(function(acc, item) {
+  return arr1.concat(arr2).reduce(function(acc, item) {
     return acc.indexOf(item) === -1
       ? acc.concat(item)
       : acc
-  }, arr1)
+  }, [])
```

Comment:
what's the reason?

Reply:
https://github.com/Kelin2025/nanoutils/pull/35#issuecomment-367583126